src/Sylius/Bundle/ShopBundle/templates/shared/product_review/product_card.html.twig

```diff
@@ -1,6 +1 @@
-{{
-    render(url('sylius_shop_partial_product_show_by_slug', {
-        'slug': app.request.attributes.get('slug'),
-        'template': '@SyliusShop/shared/product/card.html.twig'
-    }))
-}}
+{{ component('sylius_shop:common:product_card', { slug: app.request.attributes.get('slug') }) }}
```

Comment:
Can we resolve product and pass it to card component? 🤔

Reply:
all we have is slug, couldn't find anything better atm